### ISSUE TITLE
Firefox using align instead of text-align

### ISSUE THREAD

**Issue by mscheker** [jaredreich/pell#147]
It seems that the `align` attribute is not supported in HTML5. Any reason why is this applied when aligning text within the editor. Looks like in Chrome/Safari it is applying the style attribute with a text-align value, instead. Thanks for any input.

**Comment by RossMWoods** [comment]
Yes. "align" is not supported in HTML5. Should be in a style and made "text-align".
Firefox won't recognize older code if the !DOCTYPE is for html5.
But it I omit the !DOCTYPE tag, Firefox treats it as older code and renders it well.
Default is left aligned, so code is only needed if it's something else (centred, justified, right-aligned).
